packages/lit-dev-content/site/blog/2023-10-10-lit-3.0.md

```diff
@@ -0,0 +1,262 @@
+title: "Lit Launch Day: Lit 3.0, Labs graduations, a compiler and more!"
+summary: "We're launching the next major version of Lit"
+date: 2023-10-10
+author:
+  - lit-team
+---
+
+# Lit Launch Day: Lit 3.0, Labs graduations, a compiler and more!
+
+It's launch day for the Lit team, and we have a bunch of exciting releases to share with the Lit and web components communities!
+
+After several months of development, the Lit team is happy to announce the final release of Lit 3.0 - our fist major version since Lit 2.0 in early 2021 - the first graduating class of Lit Labs packages: `@lit/react`, `@lit/task`, and `@lit/context`, and two bonus releases `@lit-labs/compiler` and `@lit-labs/preact-signals`.
+
+This is a big release, so here are links to the individual announcements:
+- [Lit 3.0](#lit-3.0)
+- [The New Lit Template Compiler](#compiler)
+- [Labs Graduation Day](#labs-graduation)
+- [Preact Signals Integration](#preact-signals)
+
+## Lit 3.0: Bye, Bye IE, Hello TC39 Decorators! { #lit-3.0 }
+
+We value stability for our community. Breaking changes are a cost that every user and the entire ecosystem has to bear: projects have to upgrade, multiple versions can be included in an app, and documents, samples, tutorials, starter kits, etc. have to be updated for every breaking change.
+
+So we want to only make breaking changes when we must, or when the benefits to the community clearly outweigh the costs. For benefits we tend look for decreased code size, increased performance, reduction in maintenance burden, and better alignment with standards.
+
+We also follow [semantic versioning](https://semver.org/), so we increase the major version only when there are breaking changes. New features typically arrive in new _minor_ versions, which are backwards compatible. So Lit 3.0 is intended to only be a breaking change, with no new features. There is one exception though -standard decorators!
+
+### Breaking changes
+
+For Lit 3.0, the biggest breaking change is that we've dropped IE11 support. After surveying our developer community, we feel like now is the right time to say goodbye to IE, and very few customers will be affected.
+
+This release is also an opportunity to make a few additional breaking changes that trim out some technical debt to unlock new features we have scheduled for our 3.x release series and beyond.
+
+These changes are hopefully minor for most customers. If you run Lit 2.x with no deprecation warnings, and use toolchain that supports modern JS, this should be a seamless upgrade!
+
+Here are the biggest things Lit 3.0 changes:
+- IE11 is no longer supported.
+- Lit's npm modules are now published as ES2021.
+- APIs deprecated with the Lit 2.0 release have been removed.
+- SSR hydration support modules were moved to the `@lit-labs/ssr-client` package.
+- Decorator behavior has been unified between TypeScript experimental decorators and standard decorators.
+- Support was removed for Babel decorators version "2018-09"
+
```

Comment:
```suggestion

Full details in the [the upgrade guide](/docs/v3/releases/upgrade/).

```

Reply:
We mention it below, but it's one of the most important links in the page, it's worth repeating